src/common/history.lisp

```diff
@@ -31,15 +32,16 @@
   (and (not (equal input last-input))
        (not (equal input ""))))
 
-(defun make-history (&key pathname)
+(defun make-history (&key pathname (limit nil))
```

Comment:
style: `(limit nil)` is redundant (it's nil by default) and AFAIK not the style followed so far (eg "pathname" is nil).

Reply:
Hahah good catch, left that by accident